src/jit/jitconfigvalues.h

```diff
@@ -253,6 +253,31 @@ CONFIG_INTEGER(EnablePOPCNT, W("EnablePOPCNT"), 1)           // Enable POPCNT
 // Enable AVX instruction set for wide operations as default
 CONFIG_INTEGER(EnableAVX, W("EnableAVX"), 0)
 #endif                                                       // !defined(_TARGET_AMD64_) && !defined(_TARGET_X86_)
+
+#if defined(_TARGET_ARM64_)
+CONFIG_INTEGER(EnableArm64Aes      , W("EnableArm64Aes"), 1)
```

Comment:
Do we want these to be named `EnableArm64{Isa}`? The existing x86 ones are just named `Enable{Isa}` (although this was mostly done because of the pre-existing `EnableAVX` flag).

Reply:
I'm open to any naming scheme. "EnableArm64X" does seem odd. I could go with just "EnableX" (I thought it might be too obscure when I added the "Arm64" part), or "EnableArm64IsaX" or "EnableIsaX" or ???. Any votes?

Reply:
I think just `Enable{Isa}` is the most consistent (given the current x86 naming scheme) and given that these are architecture dependent, there shouldn't be any concern of overlap (for example, between `EnableAES` on ARM and `EnableAES` on x86)

Reply:
If we were willing to change how x86 is named (but this would be a breaking change due to `EnableAVX` being a pre-existing flag that impacts `System.Numerics.Vector<T>`), I think `Enable{Arch}{Isa}` would be a better option (so `EnableArm64{Isa}` or `EnableX86{Isa}`), as that makes it clear this is impacting architecture specific functionality.

Reply:
It's a bit unfortunate that when we (I) added `EnableAVX` it wasn't "future-proofed" for other platforms. However, I think it would be prudent to go with `EnableArm64` for these - and perhaps even for the "new" x86 flags.

The problem with overlap is that I do think it could lead to confusion - especially since we need to ensure that we recognize all variants of `IsSupported` even if it isn't relevant to the current platform, so that we can elide the dead code.